assets/content/documentation/H2d/6. Optimizing-Bitmap-Usage.md

```diff
@@ -0,0 +1,9 @@
+# Optimizing bitmaps
+
+The following are some tips for better use of bitmaps with Heaps:
+
+    * Use images with sizes of a power of 2 as textures (16, 32, 64, 128, 256 etc.). Graphics cards process textures much more quickly when images are sized as such.
+    * For fast tile rendering, use the `h2d.TileGroup` module.
+    * If you have a lot of separate bitmaps? Combine them in one texture to reduce memory usage. Software exists abundantly on the web which can pack sprites neatly into images for this purpose, including [`TexturePacker`](https://www.codeandweb.com/texturepacker).
+    * For particle rendering, use the `h2d.SpriteBatch` module.
+    * Use the `engine.drawCount` function to see the draw calls. A lower value is best.
```

Comment:
This would require more details about the need for batching (draw calls explained) and how h2d.TileGroup helps, and what is the difference between it as h2d.SpriteBatch (TileGroup has a fixed buffer, SpriteBatch reupload its buffer every frame for moving tiles)